theme/gatsby-node.js

```diff
@@ -4,8 +4,27 @@ const getPkgRepo = require('get-pkg-repo')
 const axios = require('axios')
 const uniqBy = require('lodash.uniqby')
 
+exports.sourceNodes = async ({actions, createContentDigest}) => {
```

Comment:
This doesn't need to be `async`

```suggestion
exports.sourceNodes = ({actions, createContentDigest}) => {
```